modules/web/src/app/cluster/details/cluster/rbac/template.html

```diff
@@ -14,185 +14,119 @@
 limitations under the License.
 -->
 
-<div fxLayout="row"
-     class="actions-container">
-  <div fxFlex></div>
-  <button mat-flat-button
-          color="quaternary"
-          *ngIf="isClusterRunning"
-          fxLayoutAlign="center center"
-          (click)="addBinding($event)">
-    <i class="km-icon-mask km-icon-add"></i>
-    <span>Add Binding</span>
-  </button>
-</div>
-
-<h3>Cluster</h3>
-<table class="km-table"
-       mat-table
-       matSort
-       matSortActive="name"
-       matSortDirection="asc"
-       [dataSource]="clusterBindingsDataSource"
-       [hidden]="isLoadingData(clusterBindingsDataSource.data)">>
-  <ng-container matColumnDef="kind">
-    <th mat-header-cell
-        *matHeaderCellDef
-        class="km-header-cell"
-        mat-sort-header>Kind
-    </th>
-    <td mat-cell
-        *matCellDef="let element">{{element.kind}}</td>
+<ng-container *ngIf="isClusterRunning; else loading">
+  <div fxLayout="row"
+       fxLayoutAlign="space-between center"
+       class="actions-container">
+    <div>
+      <mat-form-field fxFlex
+                      class="km-dropdown-with-suffix">
+        <mat-select class="km-select-ellipsis"
+                    [formControl]="modeControl"
+                    panelClass="preset-dropdown"
+                    disableOptionCentering
+                    required>
+          <mat-select-trigger fxLayout="row"
+                              fxLayoutGap="10px"
+                              fxLayoutAlign=" center">
+            <ng-container *ngTemplateOutlet="icons; context: {$implicit: modeControl.value}"></ng-container>
+            <span>{{modeControl.value}}</span>
+          </mat-select-trigger>
+
+          <mat-option *ngFor="let mode of modes"
+                      [value]="mode">
+            <div fxLayout="row"
+                 fxLayoutGap="10px"
+                 fxLayoutAlign=" center">
+              <ng-container *ngTemplateOutlet="icons; context: {$implicit: mode}"></ng-container>
+              <span>{{mode}}</span>
+            </div>
+          </mat-option>
+        </mat-select>
+      </mat-form-field>
+    </div>
+
+    <ng-template #icons
+                 let-mode>
+      <ng-container [ngSwitch]="mode">
+        <i *ngSwitchCase="RBACKind.ServiceAccount"
+           class="km-icon-mask km-icon-serviceaccount"></i>
+        <i *ngSwitchCase="RBACKind.Group"
+           class="km-icon-mask km-icon-group"></i>
+        <i *ngSwitchCase="RBACKind.User"
+           class="km-icon-mask km-user"></i>
+      </ng-container>
+    </ng-template>
+
+    <div [ngSwitch]="modeControl.value"
+         fxLayout="row"
+         fxLayoutAlign="space-between center"
+         fxLayoutGap="10px">
+      <ng-container *ngSwitchCase="RBACKind.ServiceAccount">
+        <button mat-flat-button
+                color="quaternary"
+                *ngIf="isClusterRunning"
```

Comment:
We can remove this `*ngIf` because its already added on parent `ng-container`. Please remove other similar `*ngIf` conditions as well.